### PR TITLE
Auto publish to PyPi

### PR DESCRIPTION
**Pull request by Nimond**
# Description

Auto publish `botx` to PyPi

# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: .github/workflows/publish-pypi.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Publish python package
+on:
+  push:
+    tags:
+      - "*.*.*"
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (vonabarak) [inline .github/workflows/publish-pypi.yml:8]:
```suggestion
    runs-on: ubuntu-20.04
```

File: .github/workflows/publish-pypi.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    steps:
+      - uses: actions/checkout@v1
```
Comment (vonabarak) [inline .github/workflows/publish-pypi.yml:10]: It is recommended to use actions/checkout@v2

File: .github/workflows/publish-pypi.yml (hunk 1 above)
Comment (Nimond) [reply .github/workflows/publish-pypi.yml]: Done

**Review by vonabarak — APPROVED** [review]

File: .github/workflows/publish-pypi.yml (hunk 2 above)
Comment (Nimond) [reply .github/workflows/publish-pypi.yml:10]: In v2  on tag event `git branch` don't show another branches

**Review by lxmnk — APPROVED** [review]

**Pull request merged**